Description: Fix regex problem: arm overrides armhf.
Author: hawken <haakonsh@gmail.com>
Bug-Debian: http://bugs.debian.org/682395

--- a/apt-mirror
+++ b/apt-mirror
@@ -253,7 +253,7 @@
 	next;
     }
 
-    if($config_line =~ /deb-(alpha|amd64|armel|arm|armhf|hppa|hurd-i386|i386|ia64|kfreebsd-i386|kfreebsd-amd64|lpia|m68k|mipsel|mips|powerpc|s390|s390x|sh|sparc)/) {
+    if($config_line =~ /^deb-(alpha|amd64|armel|arm|armhf|hppa|hurd-i386|i386|ia64|kfreebsd-i386|kfreebsd-amd64|lpia|m68k|mipsel|mips|powerpc|s390|s390x|sh|sparc)$/) {
 	push @config_binaries, [$1, @config_line];
 	next;
     }
